linchpin/provision/roles/openstack/tasks/provision_os_sg.yml

```diff
@@ -10,7 +9,7 @@
 
 - name: "provision/teardown security group"
   os_security_group_rule:
-    auth: "{{ group.0 }}"
+    # auth: "{{ group.0 }}"
```

Comment:
@14rcole You may delete this line too

Reply:
Okay, I wanted to make sure I was commenting the right one